nucleus/common/internal-api/pom.xml

```diff
@@ -129,6 +130,18 @@
             <artifactId>logging-annotation-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.14</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.weld</groupId>
+            <artifactId>weld-api</artifactId>
+            <version>2.4.Final</version>
```

Comment:
BTW if you use the man:versions plugin it will hit the version wherever it is. The version properties aren't as important as without that plugin